site/content/contribute/server/system_console.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "System Console"
+date: 2019-10-09T13:38:26-04:00
+weight: 5
+subsection: Server
+---
+
+## Adding fields to the configuration
+
+In order to add fields to the configuration you need to modify `model/config.go` in the server by adding the desired field to one of the struct such as `ServiceSettings` and setting its default value in the corresponding `SetDefaults` method.
```

Comment:
```suggestion
In order to add fields to the configuration, you need to modify `model/config.go` in the server by adding the desired field to one of the structs such as `ServiceSettings` and setting its default value in the corresponding `SetDefaults` method.
```

Reply:
Thanks for the feedback, @amyblais. Good catch! :blush: 
I just pushed the changes up.

Reply:
Thank you!